INSTALL.md

```diff
@@ -59,10 +59,20 @@ Once you install the prerequisites, you're ready to build:
 bazel build :urbit
 ```
 
-The default optimization level is `-O3`, but if you want to specify a different
-optimization level, use [`--copt`][copt]:
+If you want a debug build, which changes the optimization level of the vere
+binary from `-O3` to `-O0`, specifies to include _full_ debug information, and
+defines typically desired debug macros, e.g. "U3_CPU_DEBUG", "U3_MEMORY_DEBUG",
+etc, build the `urbitdbg` target:
 ```console
-bazel build --copt='-O0' :urbit
+$ bazel build :urbitdbg
+```
+
+If you need to specify arbitrary C compiler or linker options, use
+[`--copt`][copt] or [`--linkopt`][linkopt], respectively:
+```console
+$ bazel build --copt='-O0' :urbit
+# or on the urbitdbg target if desired
+$ bazel build --copt='-DMY_MACRO' --copt='-fno-inline-small-functions' :urbitdbg
```

Comment:
If you want to use `:urbitdbg` instead of the full `//pkg/vere:urbitdbg`, you need to add an alias to the top-level BUILD file:
```python
alias(
    name = "urbitdbg",
    actual = "//pkg/vere:urbitdbg",
)
```

Reply:
That was included in 2ed1517b0e52cc5dbf5b25eb76ef34dc2940eaa5

Reply:
Sorry, overlooked that (had the file marked as viewed when I reviewed most recently).